lib/utils/axios-helpers.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios');
+
+const auth = require('../auth');
+const config = require('../config');
+
+/**
+ * Set axios' default auth header to OAuth Bearer Token from v5 RO API
+ *
+ * @async
+ * @returns {Promise} If an error is returned from `auth.getToken` the
+ * promise will reject, otherwise it will resolve.
+ *
+ * @example
+ * await setV5Token()
+ */
+const setV5Token = () =>
+  new Promise((resolve, reject) => {
```

Comment:
@shamrt should we use node's [util.promisify](https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original)?